src/main/scala/viper/gobra/translator/encodings/preds/PredEncoding.scala

```diff
@@ -46,6 +46,7 @@ class PredEncoding extends LeafTypeEncoding {
     * (1) exclusive operations on T, which includes literals and default values
     *
     * [Q{d1, ..., dk}: pred(S)] -> make_S_ID([d1], ..., [dk]) where ID is the ID for the pattern used in Q{d1, ..., dk}
+    * [dflt(Pred(ts)°)] -> default() where default is a 0-ary function in the domain with name domainName(embedPredType(ts))
```

Comment:
`domainName(embedPredType(ts))` is not a name. But I also do not think that you have to provide the name here.